src/command_modules/azure-cli-network/azure/cli/command_modules/network/_params.py

```diff
@@ -32,15 +32,16 @@
      process_public_ip_create_namespace, validate_private_ip_address,
      process_lb_frontend_ip_namespace, process_local_gateway_create_namespace,
      process_tm_endpoint_create_namespace, process_vnet_create_namespace,
+     process_vnet_gateway_create_namespace, process_vpn_connection_create_namespace,
      validate_inbound_nat_rule_id_list, validate_address_pool_id_list,
      validate_inbound_nat_rule_name_or_id, validate_address_pool_name_or_id,
      validate_servers, load_cert_file,
-     vnet_gateway_validator, validate_peering_type,
+     validate_peering_type,
      get_public_ip_validator, get_nsg_validator, get_subnet_validator,
      get_virtual_network_validator)
 from azure.cli.command_modules.network.mgmt_nic.lib.models.nic_creation_client_enums import privateIpAddressVersion
 from azure.cli.command_modules.network.mgmt_vnet_gateway.lib.models.vnet_gateway_creation_client_enums import \
-    (gatewayType, sku, vpnType)
+    (gatewayType, sku, vpnGatewayType)
```

Comment:
Is this change expected? In the current swagger, I only see `vpnType`, which I like it better. Otherwise it is hard to tell the difference from `gatewayType`